runtime/src/lib.rs

```diff
@@ -307,8 +278,10 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-    pub const ExistentialDeposit: u128 = 500;
+    pub const ExistentialDeposit: u128 = 1 * MILLIUNIT;
     pub const MaxLocks: u32 = 50;
+    pub const MaxReserves: u32 = 50;
+    pub const TransactionByteFee: u128 = 1 * MICROUNIT;
```

Comment:
why not move this into a `parameter_types` just above `pallet_transaction_payment` like in the standalone chain since that's where it's used  https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR542

Reply:
I followed ordering according to https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.9/runtime/src/lib.rs#L250-L257

Should I make it separate for `pallet_transaction_payment`?

Regarding `TransactionByteFee`'s value, Shall I change it to `10 * MILLICENTS;` according to standalone chain?

I am not sure what is ideal value. I just followed template.

Reply:
added to #227